instagram-app/src/App.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import uuid from "uuid";
+import "./App.css";
+import dummyData from "./dummy-data.js";
+import PostsPage from "./components/PostContainer/PostsPage";
+import withAuthenticate from './components/authentication/withAuthenticate';
+import Login from './components/Login/Login';
+
+class App extends React.Component {
+  constructor() {
+    super();
```

Comment:
This is completely fine but if you are just setting state without passing any props to the constructor you have a shorter way to write it. Basically line 10 to 16 could be written as well in this fashion: 
`state = {
    data:[]
};`